src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -233,11 +233,17 @@ def get_remote_assisted_service_url(oc, namespace, service, scheme):
 
 
 def get_local_assisted_service_url(namespace, service, deploy_target):
+    res = subprocess.check_output("ip route get 1", shell=True).split()[6]
```

Comment:
please check the `res.returncode == 0`

Reply:
also if you want the host IP, it's better to use
```shell
> hostname -i
10.1.155.11
```

I think we should use python commands instead of bash

```python
import socket
print ( socket.gethostbyname(socket.gethostname()))
# output '10.1.155.11'
```